x/staking/client/rest/utils.go

```diff
@@ -28,70 +26,15 @@ func contains(stringSlice []string, txType string) bool {
 }
 
 // queries staking txs
-func queryTxs(node rpcclient.Client, cliCtx context.CLIContext, cdc *codec.Codec, tag string, delegatorAddr string) ([]sdk.TxResponse, error) {
-	page := 0
-	perPage := 100
-	prove := !cliCtx.TrustNode
-	query := fmt.Sprintf("%s='%s' AND %s='%s'", tags.Action, tag, tags.Delegator, delegatorAddr)
-	res, err := node.TxSearch(query, prove, page, perPage)
-	if err != nil {
-		return nil, err
+func queryTxs(cliCtx context.CLIContext, cdc *codec.Codec, tag string, delegatorAddr string) ([]sdk.TxResponse, error) {
```

Comment:
actually I think we can delete the staking query tx endpoint. It's a dup from regular tx query, the main difference is that you don't need to specify tags, which is silly

Reply:
^^ would be `api-breaking`

Reply:
Agreed, but let's save that for another issue/PR -- would like to avoid as much scope creep as possible. Mind opening an issue?